### ISSUE TITLE
Volume control sensitivity is all in ~5px

### ISSUE THREAD

**Issue by bunnybot** [widelands/widelands#1429]
It seems as though the volume control currently uses a linear scale, which means that in order to have any impact on the perceived volume, I need to move the controls to the bottom ~5 pixels, where a single pixel difference makes a very large change in apparent volume.

------------------------------------
Imported from Launchpad using lp2gh.

 * date created: 2013-03-03T13:23:18Z
 * owner: peter.waller
 * the launchpad url was https://bugs.launchpad.net/bugs/1142117

**Comment by bunnybot** [comment]
(by sirver)
It seems linear is sensibel still. Logarithmic seems unintuitive to me - most audio controls are linear imho. What would you expect the scale to be?

**Comment by bunnybot** [comment]
(by peter.waller)
The problem is that it's not perceptually linear. The scale should be linear in decibels, not amplitude, which I believe is the current case.

This means that the controls are almost useless, since you have no fine grained control on the apparent volume - it's all crammed into a few pixels space rather than across the entire control.

You can find more details here:

http://www.dr-lex.be/info-stuff/volumecontrols.html

**Comment by bunnybot** [comment]
(by sirver)
Thanks - this was indeed news to me. The link was useful.

**Comment by bunnybot** [comment]
(by sirver)
Setting to incomplete for bug sweeping.

**Comment by bunnybot** [comment]
(by janitor)
[Expired for widelands because there has been no activity for 60 days.]

**Comment by bunnybot** [comment]
(by gunchleoc)
Yes, we definitely should use one of the logarithmic dB scales.

**Comment by bunnybot** [comment]
[<img align='left' src='https://codeberg.org/avatars/593dc43e2444c25db9481960e5b693ab' alt='tothxa' height='48' />](https://codeberg.org/tothxa)*[Mirrored from Codeberg](https://codeberg.org/wl/widelands/issues/1429#issuecomment-8004740)*
*On Thu Oct 30 16:41:27 CET 2025, [**Tóth András (tothxa)**](https://codeberg.org/tothxa) wrote:*

---

I can't reproduce this either. I guess it must be system dependent. I'm using an integrated intel audio device with plain ALSA, without any sound daemon, and it's "perceptually linear" both with older (Debian buster) and newer (Debian trixie) SDL.
